packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -421,6 +407,58 @@ async function handleGitPreference(gitInitFlag) {
   }
 }
 
+/**
+ * @param {string?} commitMessageFlag
+ */
+async function handleCommitMessagePreference(commitMessageFlag) {
+  // Handle case where flag is set
+  if (commitMessageFlag !== null) {
+    return commitMessageFlag
+  }
```

Comment:
I didn't feel the need to draw text here like the other preference handlers

Reply:
Yeah agreed probably a bit verbose. Thoughts on displaying the initial commit message inside the `"✔ Initialised a git repo"` message? Something like `"✔ Initialised a git repo, with message 'Initial commit'"` https://github.com/redwoodjs/redwood/blob/7b83ced671c1f183108a84ef6ed2de589ee41fb2/packages/create-redwood-app/src/create-redwood-app.js#L345